var/spack/repos/builtin/packages/qthreads/package.py

```diff
@@ -20,19 +20,27 @@ class Qthreads(AutotoolsPackage):
     homepage = "http://www.cs.sandia.gov/qthreads/"
 
     url = "https://github.com/Qthreads/qthreads/releases/download/1.10/qthread-1.10.tar.bz2"
+    version("1.14", "3e6eb58baf78dc961b19a37b2dc4f9a5")
     version("1.12", "c857d175f8135eaa669f3f8fa0fb0c09")
     version("1.11", "68b5f9a41cfd1a2ac112cc4db0612326")
     version("1.10", "d1cf3cf3f30586921359f7840171e551")
 
     patch("restrict.patch", when="@:1.10")
     patch("trap.patch", when="@:1.10")
 
-    depends_on("hwloc")
+    variant(
+        'hwloc',
+        default=True,
+        description='hwloc support'
+    )
+
+    depends_on("hwloc@1.11.11", when="+hwloc")
 
     def configure_args(self):
         spec = self.spec
-        args = [
-            "--enable-guard-pages",
-            "--with-topology=hwloc",
-            "--with-hwloc=%s" % spec["hwloc"].prefix]
-        return args
+        if "+hwloc" in self.spec:
+            args = [
+                "--enable-guard-pages",
+                "--with-topology=hwloc",
+                "--with-hwloc=%s" % spec["hwloc"].prefix]
+            return args
```

Comment:
It would be helpful to have  an `else` case here to explicitly specify the other use case.  That will help make sure it doesn't use a hwloc found on the system outside of spack.

Reply:
Ok, I added in an else statement in my latest commit which explicitly disables topology support. Hwloc and guard pages don't seem to have a way to be explicitly disabled from what I could tell. It seems like just excluding those build flags is sufficient to remove support.